src/neptune/metadata_containers/model.py

```diff
@@ -265,6 +276,12 @@ def fetch_model_versions_table(self, *, columns: Optional[Iterable[str]] = None)
                     Fields: `["params/lr", "params/batch", "val/acc"]` - these fields are included as columns.
                     Namespaces: `["params", "val"]` - all the fields inside the namespaces are included as columns.
                 If `None` (default), all the columns of the model versions table are included.
+            limit: How many entries to return at most (default: None - return all entries).
+            sort_by: Name of the column to sort the results by.
+                Must be an atomic column (string, float, datetime, integer, boolean), otherwise raises `ValueError`.
+                Default: 'sys/creation_time.
```

Comment:
1. I think we should use "field" here instead of "column". In the UI, column names can be changed arbitrarily, but it has no bearing on the underlying field name.
1. See the previous comment about "atomic".
1. IMO no need to mention errors raised, as it represents a violation of the API in this case.
1. No need to mention defaults in the docstring. (We're not entirely consistent here yet; that's on me)

```suggestion
            sort_by: Name of the field to sort the results by.
                The field type must be simple (string, float, datetime, integer, or Boolean).
```

Reply:
2-4 should be done :wink:. I'm struggling with 1. as you may provide namespaces instead of just fields and the full description is quite hard for me to rephrase.

Reply:
I'd like your take on `1` (field vs column) but I can of course submit proposals directly once the PR is mature enough.